network/drivers/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Network drivers overview
+description: Overview of Docker network drivers and related concepts
+keywords: networking, drivers, bridge, routing, routing mesh, overlay, ports
+---
+
+Docker's networking subsystem is pluggable, using drivers. Several drivers
+exist by default, and provide core networking functionality:
+
+- `bridge`: The default network driver. If you don't specify a driver, this is
+  the type of network you are creating. **Bridge networks are usually used when
+  your applications run in standalone containers that need to communicate.** See
+  [bridge networks](bridge.md).
+
+- `host`: For standalone containers, remove network isolation between the
+  container and the Docker host, and use the host's networking directly. See
+  [use the host network](host.md).
+
+- `overlay`: Overlay networks connect multiple Docker daemons together and
+  enable swarm services to communicate with each other. You can also use overlay
+  networks to facilitate communication between a swarm service and a standalone
+  container, or between two standalone containers on different Docker daemons.
+  This strategy removes the need to do OS-level routing between these
+  containers. See [overlay networks](overlay.md).
+
+- `ipvlan`: IPvlan networks give users total control over both IPv4 and IPv6
+  addressing. The VLAN driver builds on top of that in giving operators complete
+  control of layer 2 VLAN tagging and even IPvlan L3 routing for users
+  interested in underlay network integration. See [IPvlan networks](ipvlan.md).
+
+- `macvlan`: Macvlan networks allow you to assign a MAC address to a container,
+  making it appear as a physical device on your network. The Docker daemon
+  routes traffic to containers by their MAC addresses. Using the `macvlan`
+  driver is sometimes the best choice when dealing with legacy applications that
+  expect to be directly connected to the physical network, rather than routed
+  through the Docker host's network stack. See
+  [Macvlan networks](macvlan.md).
+
+- `none`: For this container, disable all networking. Usually used in
+  conjunction with a custom network driver. `none` is not available for swarm
+  services. See
+  [disable container networking](none.md).
+
+- [Network plugins](/engine/extend/plugins_services/): You can install and use
```

Comment:
I just checked the preview version and it points to [this page](https://deploy-preview-17176--docsdocker.netlify.app/engine/extend/plugins_services/), which seems not so much related and pretty much unhelpful.

I remember at some point the docs had a nice list of network plugins from the community (although I think some of them were broken :sweat_smile:). Maybe we can try to re-add that. @thaJeztah Any thoughts on this?

Reply:
Are you referring to this page? https://docs.docker.com/engine/extend/legacy_plugins/#network-plugins

Reply:
Fixed the link to point to the correct page (networking plugins)